en/docs/learn/outbound-provisioning.md

```diff
@@ -5,19 +5,19 @@ identity provider from the WSO2 Identity Server. A trusted identity
 provider is basically an identity provider that supports inbound
 provisioning. It can be Google, Salesforce, another Identity Server,
 etc. For more information, see [Outbound
-provisioning](Identity_Provisioning).
+provisioning](identity-provisioning.md).
```

Comment:
this should change to ../../learn/identity-provisioning

please change all links to the above mentioned format